python/samples/concepts/agents/assistant_agent_chart_maker.py

```diff
@@ -17,7 +18,9 @@
 AGENT_INSTRUCTIONS = "Create charts as requested without explanation."
 
 # Note: you may toggle this to switch between AzureOpenAI and OpenAI
-use_azure_openai = True
+use_azure_openai = False
```

Comment:
nit: Can we keep this the unchanged?

Reply:
Definitely! Will revert. Thanks for catching.